packaging/debian/control

```diff
@@ -1,36 +1,39 @@
-Source: google-compute-engine
+Source: gce-compute-image-packages
+Maintainer: Google Cloud Team <gc-team@google.com>
 Section: admin
 Priority: optional
-Maintainer: Google Cloud Team <gc-team@google.com>
 Build-Depends: debhelper (>= 10)
 Standards-Version: 3.9.8
 Homepage: https://github.com/GoogleCloudPlatform/guest-configs
 
 Package: google-compute-engine
 Architecture: all
-Depends: google-compute-engine-oslogin,
+Depends: ${misc:Depends},
+         google-compute-engine-oslogin (>> 20190801),
          google-guest-agent,
          nvme-cli,
          ${misc:Depends}
-Recommends: rsyslog | system-log-daemon
 Provides: irqbalance
-Conflicts: google-compute-engine-jessie,
-           google-compute-engine-init-jessie,
-           google-config-jessie,
-           google-compute-engine-stretch,
-           google-compute-engine-init-stretch,
-           google-config-stretch,
-           google-compute-daemon,
-           google-startup-scripts,
-           irqbalance
-Replaces: google-compute-engine-jessie,
-          google-compute-engine-init-jessie,
-          google-config-jessie,
-          google-compute-engine-stretch,
-          google-compute-engine-init-stretch,
-          google-config-stretch,
-          google-compute-daemon,
-          google-startup-scripts
+Recommends: rsyslog | system-log-daemon
+Suggests: libpam-cracklib
+Conflicts: gce-cloud-config,
```

Comment:
This is missing all of the older Debian `Conflicts` and `Replaces`, like the `*-(jessie|stretch)` stuff.

Reply:
Sounds good, I didn't think of that.